test/UnitTest/Microsoft.AspNet.OData.Test.Shared/OrderByTest.cs

```diff
@@ -0,0 +1,139 @@
+﻿using Microsoft.AspNet.OData.Builder;
+using Microsoft.AspNet.OData.Extensions;
+using Microsoft.AspNet.OData.Test.Abstraction;
+using Microsoft.OData.Edm;
+using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+using System.Net.Http;
+using System.Net.Http.Headers;
+using System.Threading.Tasks;
+using Xunit;
+using System.Linq;
+
+namespace Microsoft.AspNet.OData.Test
+{
+    public class OrderByTest
+    {
+        [Fact]
+        public async Task OrderBy_MultipleLevelOrderByTest()
```

Comment:
>OrderBy_MultipleLevelOrderByTest [](start = 26, length = 32)

Such test case is better to put in the e2e

Reply:
This file was removed